src/main.js

```diff
@@ -1,32 +1,144 @@
-import { Worker as NodeWorker } from 'worker_threads';
-import { createRequire } from 'module';
+import { Worker as NodeWorker } from "worker_threads";
+import { createRequire } from "module";
+import { initialzieCqlWorker } from "cql-worker";
```

Comment:
I noticed you're changing all of the single quotes to double quotes in the files that you're editting. Is this intentional, or done because of a linter you're using? If we are considering changing all single quotes to double quotes, I wonder if we should do it in a separate PR, since doing it automatically in just the files that you're editting makes the quote style inconsistent throughout the codebase and makes the diffs pretty messy to read.

Reply:
That was done by a linter.  I dont know if I can change everything that it changed.

Reply:
Formatting changes, including quote changes, have been removed in https://github.com/ccsm-cds-tools/encender/pull/15/commits/891bfbdc776bd725b439903ead71b6f733ff423e.